stock_picking_move_done_manual/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,14 @@
+To configure this module, you need to:
```

Comment:
I think this does not belong here

Reply:
@AdriaGForgeFlow Thanks. Comment attended.